stories/Display/TabsStory.tsx

```diff
@@ -15,28 +19,91 @@ const props = {
       require: 'yes',
       description: 'Sets title of Tab',
     },
+    {
+      name: 'alignment',
+      type: 'string',
+      defaultValue: 'horizontal',
+      possibleValue: 'horizontal, vertical',
+      require: 'yes',
+      description: 'Sets alignment of Tab',
+    },
+    {
+      name: 'variant',
+      type: 'string',
+      defaultValue: 'underlined',
+      possibleValue: 'underlined, colored',
+      require: 'yes',
+      description: 'Set the change in the color pattern only for Horizontal tabs',
+    },
   ],
 };
 
 const TabsStory = () => (
-  <StorybookComponent
-    title="Tabs"
-    code="import { Tabs } from 'glints-aries'"
-    propsObject={props}
-    usage={`<Tabs>
-  <Tabs.Pane tab="Location Tab">Tab Location</Tabs.Pane>
-  <Tabs.Pane tab="Salary">Tab Salary</Tabs.Pane>
-  <Tabs.Pane tab="Job">Tab Job</Tabs.Pane>
-  <Tabs.Pane tab="Company">Tab Company</Tabs.Pane>
-</Tabs>`}
-  >
-    <Tabs>
-      <Tabs.Pane tab="Job">Software Engineer <Badge label="1" /></Tabs.Pane>
-      <Tabs.Pane tab="Company">Glints</Tabs.Pane>
-      <Tabs.Pane tab="Location">Jakarta</Tabs.Pane>
-      <Tabs.Pane tab="Salary">Rp 10,000,000</Tabs.Pane>
-    </Tabs>
-  </StorybookComponent>
+  <React.Fragment>
+    <StorybookComponent
+      title="Tabs"
+      code="import { Tabs } from 'glints-aries'"
+      propsObject={props}
+      usage={`<Tabs alignment="horizontal" variant="underlined">
+        <Tabs.Pane tab="Location Tab">Tab Location</Tabs.Pane>
```

Comment:
There is an error here `Property 'Pane' does not exist on type 'FunctionComponent<Props>'`. You'll need to cut this code from `NavigationTabs.tsx` and paste it on `Tabs.tsx`. 

```
type Tabs = React.FunctionComponent<Props> & {
  Pane: typeof TabPane;
}

Tabs.Pane = TabPane;
```

Reply:
@franciscrispin, I just realised that the file `NavigationTabs.tsx` is not needed now and the code can simply be added to `Tabs.tsx` as only the classNames are being changed.

Reply:
OH YES. You are right haha :+1: